_pages/workshop.md

```diff
@@ -32,7 +32,8 @@ The goal of this workshop series is to review the classification of terms in Mon
 The emerging agenda is available [here](https://docs.google.com/document/d/1Cg6fSNXaC3PZGIEnWHq_R8InKkQ1J2wVgkpbbxHhLZY/edit). To RSVP, please fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSd2vJU6XoUPmOC2S-YNcO4wuQOKrSo4YjGcxKFNrbXmGUybaA/viewform).
 
 ### Recording
-The recording will be shared after the event.
+Part 1 [recording here](https://www.youtube.com/@monarchinitiative/videos)
```

Comment:
@nicolevasilevsky do we have a more specific link directly to the actual video? This link it for the youtube channel.

Reply:
oh yeah, there is a youtube link, that is what I meant to add! I'll fix it

Reply:
updated @sabrinatoro